backend/btrixcloud/crawlconfigs.py

```diff
@@ -1220,6 +1226,21 @@ async def run_now_internal(
             print(traceback.format_exc())
             raise HTTPException(status_code=500, detail=f"Error starting crawl: {exc}")
 
+    async def too_many_waiting_crawls_check(self, org: Organization):
+        """if max concurrent crawls are set, limit number of queued crawls to X concurrent limit
+        return 429 if at limit"""
+        max_concur = org.quotas.maxConcurrentCrawls
+        if not max_concur:
+            return False
```

Comment:
Since we're not doing anything with these return values and we'll never return true, could just have an empty return if we're good?